test/window.translations.js

```diff
@@ -0,0 +1,8 @@
+const translations = {};
+window = {
```

Comment:
Is window declared but empty?

Reply:
No, it was an undefined var in the node context.

Reply:
So you don't want to declare it?

Reply:
If we declare it, it won't work.

In the js module system, doing:
```
window = {}
```
Will make a global variable available to every other module.

While:
```
const window = {};
```
(or var or let) will make a local variable only available to the current module.